### PR TITLE
Finish list editing

### PR DESCRIPTION
**Pull request by ZaphodBeebblebrox**
For some reason, I decided to look through the finnish list and see if I can help it at all. I don't speak finish, I was just making edits to make the list closer to being a good filter list. 99% of the edits I made should not effect the intended purpose of the original author, but simply make them more efficent and/or less prone to side effects. If you want, just reject this.

What I did:
I first changed network filters with a domain name to include a `||` if they did not have one.
Next, I ran it through the redundancy checker alongside easylist and removed filters that were made a duplicate by easylist.
Then I added types to relevant network filters (ex: if the file ended with.png or the path contained image, I added `$image`, if it ended in .js I added `$script` etc.
Finally, I removed/ changed a few bad cosmetic filters. For these, I checked the sites they were on to see if they were actually doing anything, and most of them did not actually target anything on any of the pages I went to on each site.

### CODE REVIEW

**Comment by DandelionSprout** [comment]
I know by now that you're something of an expert on syntaxes. And considering that [the source list for ≥90% of the original entries](https://raw.githubusercontent.com/wiltteri/subscriptions/master/wiltteri.txt) has been unmaintained since 2014, I give a lot of leeway into attempting to modernise it somewhat.

Therefore I accept these changes. 🏅

**Comment by ZaphodBeebblebrox** [comment]
Honestly, what this list really needs is someone to burn it and start over, but there is no real maintainer at the moment. The list is full of old rules that no longer apply to the sites that they are made for, but slogging through all of that is way more of a time commitment than I, or probably anyone else, is willing to do.

**Comment by DandelionSprout** [comment]
My Finnish skills are highly insufficent when it comes to actually browsing through the sites, so I'd be surprised if I was able to do it. After all, I decided to write the title of the list in Swedish (A traditional minority language in Finland, previously an upperclass language), as I couldn't even make a guess at how to formulate and correctly bend a 4-word sentence in Finnish.

Getting Juvander to do it, would be out of the question. My best bet is to add the makeshift list to uBO, while keeping the note about `(Reliable maintainers wanted)` in the title, and hoping that any sort of Finnish or Finland-related geek out there heeds the call of duty.

File: AncientLibrary/Dandelion Sprout's Finnish List.txt [hunk 1]
```diff
@@ -3952,9 +3944,9 @@ yyterintwist.fi##td + td + td > center
 ||ylilauta.org/img/suomicasino.gif$image
 ||ylilauta.org/img/synnit*.jpg$image
 ||ylilauta.org/img/ylilauta$image
-||yllas.fi/sites/default/files/*150x60$image,subdocuemnt
-||yllas.fi/sites/default/files/*201x77$image,subdocuemnt
-||yllas.fi/sites/default/files/*490x80$image,subdocuemnt
+||yllas.fi/sites/default/files/*150x60$image,subdocuement
+||yllas.fi/sites/default/files/*201x77$image,subdocuement
+||yllas.fi/sites/default/files/*490x80$image,subdocuement
```
Comment (okiehsch) [inline AncientLibrary/Dandelion Sprout's Finnish List.txt]: `subdocuemnt` ->`subdocuement` -> `subdocument`

Comment (DandelionSprout) [reply AncientLibrary/Dandelion Sprout's Finnish List.txt]: Well spotted. 😅

**Pull request merged**